index.js

```diff
@@ -78,13 +80,16 @@ function Filter(inputTree, options) {
     if (options.persist) {
       this.processor.setStrategy(require('./lib/strategies/persistent'));
     }
+    this.async = (options.async === true);
   }
 
   this.processor.init(this);
 
   this._canProcessCache = Object.create(null);
   this._destFilePathCache = Object.create(null);
   this._needsReset = false;
+
+  this.concurrency = Number(process.env.JOBS) || require('os').cpus().length;
```

Comment:
`this._concurrency` as it is not public API yet, unless it should be public API and then the readme should make reference to it.

If we want to expose it, we should likely do

```js
this.concurrency = options.concurrency || Number(process.env.JOBS) || require('os').cpus().length;
```